src/TTextEdit.cpp

```diff
@@ -1431,27 +1480,43 @@ void TTextEdit::slot_copySelectionToClipboardImage()
     }
 
     // Qt says: "Maximum supported image dimension is 65500 pixels" in stdout
-    auto heightpx = qMin(65500, (mPB.y() - mPA.y() + 1) * mFontHeight);
+    auto heightpx = std::min(65500, (mPB.y() - mPA.y() + 1) * mFontHeight);
     auto lineOffset = mPA.y();
 
     // find the biggest width of text we need to work with
     int largestLine{};
+    qDebug().noquote().nospace() << "TTextEdit::slot_copySelectionToClipboardImage() INFO - line widths in \"normal\" character widths (and pixels):";
```

Comment:
This debug information is not necessary, I guess this is a testing left over?

Reply:
Yeah - and it did lead me to a bug I had made - it can probably be gotten rid of along with the other things...

Reply:
... will be gone in next commit.